src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -5061,39 +5061,40 @@ Real64 DownInterpolate4HistoryValues(Real64 OldTimeStep, Real64 NewTimeStep, std
     Real64 const newTime1 = newTime0 - NewTimeStep;
     Real64 const newTime2 = newTime1 - NewTimeStep;
     Real64 const newTime3 = newTime2 - NewTimeStep;
-    Real64 const newTime4 = newTime3 - NewTimeStep;
```

Comment:
because all of the `newVals` are moved back by one (i.e. `newVals[1]` is equal to what used to be `newVals[0]`), newTime4 is no longer used.